packages/discord.js/src/structures/DirectoryChannel.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Channel } = require('./Channel');
+
+/**
+ * Represent a channel that displays a directory of guilds
+ */
+class DirectoryChannel extends Channel {
```

Comment:
Aren't directory channels, guild channels?

Reply:
I don't believe so. Additionally, we wouldn't be able to use any of the methods a guild channel offers as the only way to receive this is to fetch a hub invite.

Reply:
Basically what Jira said